avocado/utils/cloudinit.py

```diff
@@ -174,21 +174,33 @@ def __init__(self, address, instance_id):
         self.instance_id = instance_id
         self.instance_phoned_back = False
 
+    @classmethod
+    def setup_and_wait_for_phone_home(cls, address, instance_id):
```

Comment:
Because it's an action/verb, it should be `set_up`, and not `setup` (this is a very common mistake of mine too).